src/plugin/includes/class-podlove-web-player-embed-api.php

```diff
@@ -278,4 +282,14 @@ public function config(WP_REST_Request $request)
 
         return rest_ensure_response(array_merge($config, $theme, $share));
     }
+  
+    /**
+     * Check API permissions
+     *
+     * @since    5.3.1
+     */
+    public function api_permissions()
+    {
+      return current_user_can('manage_options');
```

Comment:
This API is meant to be used without authentication :D So I guess `permission_callback => '__return true;'` should be fine.